CHANGELOG.md

```diff
@@ -3,6 +3,117 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 5.6.0
+
+_Mar 3, 2022_
+
+A big thanks to the 15 contributors who made this release possible. Here are some highlights ✨:
+
+- 📦 Use the bundling scripts as the packages published by the [https://github.com/mui/material-ui](material-ui) repository (#3965) @flaviendelangle
+
+  The code structure and the bundling strategy have been modified to provide a better isolation between components.
+  This brings tree shaking, allowing to reduce the bundle size. 🏋
+  Such modification is always a bit risky.
+  If you encounter problems with your build, please open an issue.
+  These issues will have high priority.
+
+- 🧼 Clean and document the columns selector (#4010) @flaviendelangle
+
+  To improve clarity, columns selectors have been renamed.
+  Old naming are deprecated, and will be removed in v6.
+  Here is the renaming and the modification needed to get same information with new selectors.
+
+  |Previews naming|New naming|  
+  |---|---|
```

Comment:
Could you format the table extending the width of the largest column? Although it works on GitHub UI, it doesn't look good when reading the CHANGELOG in a text editor. https://www.tablesgenerator.com/markdown_tables helps.

Reply:
I used https://markdownbeautifier.com 

But I'm surprised that even I put the table in the `docs` folder it is not pretiffied by `yarn prettier`